src/util.js

```diff
@@ -90,6 +85,14 @@ const defaultMaxSize = 64 * 1024 * 1024 // max heap size when auto-growing, 64 M
 let maxSize = defaultMaxSize
 let decoder = null
 
+/**
+ * Configure the underlying CBOR decoder.
+ *
+ * @param {Object} [options] - The options the decoder takes. The decoder will reset to the defaul values if no options are given.
```

Comment:
Tiny nit depending on how picky you're feeling: check your @param blocks through these changes, some have `.` on the end and others don't. This one has multiple sentences so is justifiable but some others without multiple sentences have them and some don't.

Reply:
I tried to be consistent, but apparently I wasn't (thanks for mentioning this, I'm that nitpicky myself :)